sections/main-product.liquid

```diff
@@ -252,7 +252,7 @@
               </div>
             </noscript>
           {%- when 'buy_buttons' -%}
-            <product-form class="product-form">
+            <product-form class="product-form" {{ block.shopify_attributes }}>
```

Comment:
This misses `<pickup-availability>` but maybe ok?

Reply:
https://github.com/Shopify/dawn/pull/68#discussion_r660959217